php/commands/src/CLI_Command.php

```diff
@@ -575,4 +575,38 @@ private function get_update_type_str( $assoc_args ) {
 		return $update_type;
 	}
 
+	/**
+	 * Detects if a command exists
+	 *
+	 * This commands checks if a command is registered with WP-CLI.
+	 * If the command is found then it returns with exit status 0.
+	 * If the command doesn't exist, then it will exit with status 1.
+	 *
+	 * ## OPTIONS
+	 * <command_name>...
+	 * : The command
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     The "site delete" command is registered.
```

Comment:
Fir the examples, you should follow the notation convention, as @danielbachhuber had provided:

Comments start with `#`, and the command starts with the "basic" prompt `$`. Lines with either are output.

Here's one example:

```
/**
 * ## EXAMPLES
 *
 * # The "site delete" command is registered.
 * $ wp cli has-command "site delete"
 * $ echo $?
 * 0
 */
```

Reply:
Here's a link to the documentation for this: https://make.wordpress.org/cli/handbook/documentation-standards/#command-annotation